.golangci.yml

```diff
@@ -40,6 +40,14 @@ linters-settings:
       - suite-extra-assert-call
 
 issues:
+  # Maximum issues count per one linter.
+  # Set to 0 to disable.
+  # Default: 50
+  max-issues-per-linter: 0
+  # Maximum count of issues with the same text.
+  # Set to 0 to disable.
+  # Default: 3
+  max-same-issues: 0
```

Comment:
So that the linter doesn't skip the issues if there are multiple, it could be annoying sometimes to push a change and figure out there were more linter issues, that were not showed (from personal experience 😅)